### PR TITLE
chore: add contribution notice with GitHub link

### PR DESCRIPTION
**Pull request by praisejahuzoma**
This commit updates the footer in the simplistic frontend HTML files to include a contribution notice. The notice encourages users to check out the Sleep API on GitHub to contribute to the project.

### CODE REVIEW

**Review by SleepAPI — CHANGES_REQUESTED** [review]
Unfortunately there's no generic footer component in our very simplistic frontend. This means the footer functionality has to be duplicated across all html files, there's like 7 of them. Such as production-calculator.html.

Since the pipeline failed on on prettier check it didn't get to run the conventional commit lint, but I'm guessing there's a good chance it won't accept the emoji in commit header.

File: backend/src/assets/index.html [hunk 1]
```diff
@@ -37,8 +37,10 @@ <h3>You'll find all available rankings in the menu above</h3>
       <img src="/cook.gif" alt="He who cooks" title="Who let bro cook" />
     </p>
 
-    <footer class="w3-container w3-black">
-      <a href="https://discord.gg/w97qFff8n4">Sleep API authors</a> on Discord for questions
-    </footer>
+   <footer class="w3-container w3-black">
```
[the remark this answers is not in the document]
Comment (praisejahuzoma) [reply backend/src/assets/index.html]: Ok
please check the fix commit

**Comment by praisejahuzoma** [comment]
Ok I would so that now
sorry for not seeing this

**Review by SleepAPI — CHANGES_REQUESTED** [review]
Links look good now, just missing the commit squash.
If you're unsure how to do that, one way of doing it from command line is:
```css
git rebase -i HEAD~4
```

After that you'll get an interactive editor pop-up which asks you what to do with the commits. You'll leave pick for one commit, I suggest for: `[chore: update contribution notice in HTML files]`, for the rest you change pick to squash. So it'll look something like:

```sql
squash abc123 First commit message
squash def456 Second commit message
pick ghi789 Third commit message
squash jkl012 Fourth commit message
```

Then save and close the editor. `git status` should now say something like 1 and 4 commits diff. You now have squashed your commits into one commit and you can force push it up.

**Comment by praisejahuzoma** [comment]
Sorry for bothering you a lot, this is my first time squashing a commit,
please check if I have done it right now

**Comment by SleepAPI** [comment]
> Sorry for bothering you a lot, this is my first time squashing a commit, please check if I have done it right now

Don't worry about bothering me, I'm not in a rush :)

Looking at the history of your pushes it looks like you pushed 2 new commits to the PR first and then force pushed a squash:
image

If you look at the commits currently attached to this PR there's 3 commits, we're trying to go to just 1 commit:
image

I'm not sure what caused you to push 2 commits to the PR, but that would put the total at 6 commits, then you pushed a squash which squashed 4 of those to 1 commit. Finally landing at 3 commits. So judging by that the squash probably went successful, but something caused you to push 2 commits first.

So now you 3 commits on the PR and need to squash those 3 commits into 1 commit. You can follow the directions from my previous comment, but change the amount to squash to 3 instead of 4 like:
```css
git rebase -i HEAD~3
```

Please don't hesitate to ask questions if you have any 👍

**Comment by praisejahuzoma** [comment]
Ok thanks
so I should squash just one which is this one `[chore: fix GitHub link in contribution notice]`

**Comment by praisejahuzoma** [comment]
You check it out now

**Review by SleepAPI — APPROVED** [review]
Looks great, approved! Thank you for this!
I'll merge it right away 👍

**Comment by praisejahuzoma** [comment]
I can go ahead and delete the breach right?

**Comment by SleepAPI** [comment]
> Thank you so much for Being my mentor and guiding me to finish this project.
You are the best teacher and patient person I have met.
That's very kind of you, it's you I should thank for your contribution 🙏 

> I can go ahead and delete the branch right?
Sleep API project is set up to automatically delete branches after merge, so it's already been done. 
You're all done, I'll close the ticket 👍

**Pull request merged**